finish/src/main/java/io/openliberty/guides/cart/CartApplication.java

```diff
@@ -12,12 +12,12 @@
  *******************************************************************************/
  // end::copyright[]
 
-package io.openliberty.guides.sessions;
+package io.openliberty.guides.cart;
 
 import javax.ws.rs.ApplicationPath;
 import javax.ws.rs.core.Application;
 
-@ApplicationPath("/sessions")
-public class SessionApplication extends Application {
+@ApplicationPath("/")
```

Comment:
Does removing this change the URL that needs to be accessed when using OpenAPI or the tests?

Reply:
Yes, I'll update those accordingly.